docs/tutorials/05-external-jsonschema.md

```diff
@@ -208,71 +208,73 @@ deployExecutions:
+
+      updateStrategy: patch
+
+      manifests:
+        - policy: manage
+          manifest:
+            apiVersion: apps/v1
+            kind: Deployment
             metadata:
-              labels:
-                app: echo-server
+              name: {{ $name }}
+              namespace: {{ $namespace }}
             spec:
-              containers:
-                - image: {{ with (getResource .cd "name" "echo-server-image") }}{{ .access.imageReference }}{{end}}
-                  imagePullPolicy: IfNotPresent
-                  name: echo-server
-                  args:
-                  - -text="hello world"
-                  ports:
-                    - containerPort: 5678
-                  resources:
-{{ toYaml .imports.resources | indent 21 }}
-    - policy: manage
-      manifest:
-        apiVersion: v1
-        kind: Service
-        metadata:
-          name: {{ $name }}
-          namespace: {{ $namespace }}
-        spec:
-          selector:
-            app: echo-server
-          ports:
-          - protocol: TCP
-            port: 80
-            targetPort: 5678
-      - apiVersion: networking.k8s.io/v1
-        kind: Ingress
-        metadata:
-          name: {{ $name }}
-          namespace: {{ $namespace }}
-          annotations:
-            nginx.ingress.kubernetes.io/rewrite-target: /
-            kubernetes.io/ingress.class: "{{ .imports.ingressClass }}"
-        spec:
-          rules:
-          - http:
-              paths:
+              replicas: 1
+              selector:
+                matchLabels:
+                  app: echo-server
+              template:
+                metadata:
+                  labels:
+                    app: echo-server
+                spec:
+                  containers:
+                    - image: {{ with (getResource .cd "name" "echo-server-image") }}{{ .access.imageReference }}{{end}}
+                      imagePullPolicy: IfNotPresent
+                      name: echo-server
+                      args:
+                        - -text="hello world"
+                      ports:
+                        - containerPort: 5678
+                      resources:
+  {{ toYaml .imports.resources | indent 21 }}
+- policy: manage
+  manifest:
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: {{ $name }}
+      namespace: {{ $namespace }}
+    spec:
+      selector:
+        app: echo-server
+      ports:
+        - protocol: TCP
+          port: 80
+          targetPort: 5678
+- policy: manage
+  manifest:
+    apiVersion: networking.k8s.io/v1
+    kind: Ingress
+    metadata:
+      name: {{ $name }}
+      namespace: {{ $namespace }}
+      annotations:
+        nginx.ingress.kubernetes.io/rewrite-target: /
+        kubernetes.io/ingress.class: "{{ .imports.ingressClass }}"
+    spec:
+      rules:
+        - http:
+            paths:
               - path: /
                 pathType: Prefix
                 backend:
```

Comment:
I think it would be better to also use the variable $name for the service name here, otherwise things could easily get misconfigured

Reply:
@jschicktanz I'm absolutely OK with that. However the blueprint template in this tutorial is based on the template used in tutorial number 02. I would rather update both of them, otherwise it might be confusing. Would it be ok to do that in a seperate PR?

Reply:
Yes, good point to correct this bad smell also in other places if it exists there.  

For me both is ok, you can add it to this PR or create a new one

Reply:
Done within this PR